docs/data-sources/apigateway_route.md

```diff
@@ -0,0 +1,62 @@
+---
+subcategory: "ApiGateway"
+layout: "ionoscloud"
+page_title: "IonosCloud: route"
```

Comment:
I dont know if it matters, but this might need to be `IonosCloud: ionoscloud_apigateway_route`

Reply:
changed it in https://github.com/ionos-cloud/terraform-provider-ionoscloud/pull/601